lib/sequencescape/barcode_printer.rb

```diff
@@ -1,7 +1,7 @@
 require 'sequencescape-api/resource'
 
 class Sequencescape::BarcodePrinter < Sequencescape::Api::Resource
-  attribute_reader :name, :active
+  attribute_reader :name, :active, :print_service
```

Comment:
I guess we will also want to increment the version number, as it says in the readme?

Reply:
Haven't ran `rake release` yet
Does SS and other apps (Limber/ Lighthouse UI?) using this client/gem need to have `bundle update sequencescape-client-api` ran or their version numbers updated?

Reply:
Lighthouse UI won't be able to use it as it is a ruby gem but for SS or limber you can do bundle update or change the gemfile.